ThreadingTimer.py

```diff
@@ -0,0 +1,22 @@
+import threading
```

Comment:
ThreadingTimer.py should be put somewhare like lib or src. I am fine to keep it is here but we need to refine this in the future.